src/AppInstallerCLITests/SQLiteIndex.cpp

```diff
@@ -1873,6 +1932,11 @@ TEST_CASE("SQLiteIndex_Search_VersionSorting", "[sqliteindex]")
 
     TestPrepareForRead(index);
 
+    if (!AreChannelsSupported(index))
+    {
+        return;
+    }

```

Comment:
You repeat this a few times. Would a `RETURNIF(condition , [return value])` macro be useful to reduce code at all?

Reply:
I don't feel like that is worth the additional cognitive load of knowing what it does to save the lines.